### PR TITLE
Canvas preview (lighter DOM, no pixel glitches)

### PR DESCRIPTION
**Pull request by laurent-h**

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -439,11 +434,11 @@ button:active {
...
 }
 
-.Preview__Pixel {
-  /*float: left;*/
+.Preview__Canvas {
+  display: block;
   position: relative;
-  width: var(--pixel);
-  height: var(--pixel);
+  width: calc(var(--pixel) * 8);
```
Comment (xat) [inline src/index.css:440]: is it needed to set the width/height here because it is already set to 100% inline.

Comment (laurent-h) [reply src/index.css:440]: No need for this indeed. 
position:relative is not needed either.

Comment (laurent-h) [reply src/index.css:440]: I'm not sure what is the best way to fix canvas width and height in grid and large preview, especially height.

**Comment by laurent-h** [comment]
The border seems to come from the glitch-fix CSS hack. It's probably not needed with a canvas.

**Comment by laurent-h** [comment]
I moved inline CSS into index.css and removed the :before glitch fix. There is no border on firefox anymore on my side.

**Comment by laurent-h** [comment]
Resolved conflicts.

**Comment by xat** [comment]
@laurent-h thank you very much!

i'll merge it now and we will do some small adjustments.

**Pull request merged**